src/sentry/lang/javascript/processor.py

```diff
@@ -381,6 +391,17 @@ def fetch_file(url, project=None, release=None, dist=None, allow_scraping=True):
     return result
 
 
+def get_max_age(headers):
+    cache_control = 'cache-control'
```

Comment:
I'm being a bit picky here, but this variable seems not very useful to me. It's literally just a variable for a constant which doesn't even make it shorter. I'd say you can work around this by doing like:

```python
cache_control = headers.get('cache-control')
if cache_control:
    match = ...
```

Instead which would remove the need to use the variable twice and then make the variable the value of the header rather than the key name.